pkg/ddl/reorg.go

```diff
@@ -620,8 +634,7 @@ func buildCommonHandleFromChunkRow(sctx *stmtctx.StatementContext, tblInfo *mode
 	tablecodec.TruncateIndexValues(tblInfo, idxInfo, datumRow)
 
 	var handleBytes []byte
-	handleBytes, err := codec.EncodeKey(sctx.TimeZone(), nil, datumRow...)
-	err = sctx.HandleError(err)
+	handleBytes, err := codec.EncodeKey(loc, nil, datumRow...)
```

Comment:
Seems the HandleError takes no effect here because `ErrGroupTruncate` is set to `LevelError` in  `sctx.ErrCtx()`. Do we not need `HandleError` here